package.json

```diff
@@ -53,14 +53,14 @@
     "grunt-contrib-copy": "^0.7.0",
     "grunt-contrib-jshint": "^0.11.2",
     "grunt-contrib-uglify": "^0.7.0",
-    "grunt-jsdoc": "^0.5.8",
+    "grunt-jsdoc": "^2.0.0",
```

Comment:
after the update, running `grunt docs` will fail with the following message:
```
>> Cannot parse the config file jsdoc.conf.json: SyntaxError: Unexpected token }
>> Exiting JSDoc because an error occurred. See the previous log messages for details.
Warning: jsdoc terminated with a non-zero exit code Use --force to continue.
```